aten/src/ATen/core/function_schema.h

```diff
@@ -146,12 +154,17 @@ struct FunctionSchema {
     return is_varret_;
   }
   bool is_mutable() const {
-    return std::any_of(
-        arguments_.cbegin(), arguments_.cend(), [](const Argument& arg) {
-          const auto& aliasInfo = arg.alias_info();
-          return aliasInfo && aliasInfo.value().isWrite();
-        });
+    return has_alias_annotation_ ||
+        std::any_of(
```

Comment:
Above has_alias_annotation says: `False if this schema was inferred for a custom operator`
Is this supposed to be `return !has_alias_annotation  || ...`

Reply:
sorry I was updating this a bunch. It should be `!has_alias_annotation`.